src/extensions/fixtures/fixtures_loader.js

```diff
@@ -11,183 +11,214 @@ const chalk = require('chalk')
 const yaml = require('js-yaml')
 const _ = require('lodash')
 
-let fixturesDir = 'fixtures'
-let featureUri
-
 /**
- * Configures the loader
+ * Fixtures loader extension.
  *
- * @param {string} _fixturesDir - The name of the fixtures directory relative to feature
+ * @class
  */
-exports.configure = ({ fixturesDir: _fixturesDir = 'fixtures' } = {}) => {
-    fixturesDir = _fixturesDir
-}
+class FixturesLoader {
+    /**
+     * @param {string} _fixturesDir - The name of the fixtures directory relative to feature
+     */
+    constructor({ fixturesDir: _fixturesDir = 'fixtures' } = {}) {
+        this.fixturesDir = _fixturesDir
+        this.featureUri = undefined
+    }
 
-/**
- * Sets feature uri, used to resolve fixtures files.
- * When trying to load a fixture file the path will be comprised of:
- * - feature uri
- * - fixturesDir
- * - fixture name
- *
- * @param {string} _featureUri - Feature uri
- */
-exports.setFeatureUri = _featureUri => {
-    featureUri = _featureUri
-}
+    /**
+     * Configures the loader
+     *
+     * @param {string} _fixturesDir - The name of the fixtures directory relative to feature
+     */
+    configure({ fixturesDir: _fixturesDir = 'fixtures' } = {}) {
+        this.fixturesDir = _fixturesDir
+    }
 
-/**
- * Loads content from file.
- *
- * @param {string} file - File path
- * @return {Promise.<string>} File content
- */
-exports.loadText = file =>
-    new Promise((resolve, reject) => {
-        fs.readFile(file, (err, data) => {
-            if (err) return reject(err)
-            resolve(data.toString('utf8'))
+    /**
+     * Sets feature uri, used to resolve fixtures files.
+     * When trying to load a fixture file the path will be comprised of:
+     * - feature uri
+     * - fixturesDir
+     * - fixture name
+     *
+     * @param {string} _featureUri - Feature uri
+     */
+    setFeatureUri(_featureUri) {
```

Comment:
You can remove the `_`, it was only there to not conflict with var name which is now bound to `this`

Reply:
ok